src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -46,6 +44,36 @@ public PrimitiveColumn(string name, ReadOnlyMemory<byte> buffer, ReadOnlyMemory<
             _columnContainer = new PrimitiveColumnContainer<T>(buffer, nullBitMap, length, nullCount);
         }
 
+        /// <summary>
+        /// Returns an enumerable of mutable memory buffers representing the underlying values
+        /// </summary>
+        /// <remarks>Null values are represented as default(T)</remarks>
```

Comment:
I'm now returning the null bit maps in a ReadOnlyMemory<byte>. There's a comment in the remarks that says that the bit maps are in the Apache Arrow format, so anyone could read the Arrow docs to learn how to manipulate the bits. The reason it is a ReadOnlyMemory instead of a Memory is that the `NullCount` book-keeping is done at the column level, not at the buffer level. If we expose a `Memory<byte>`, we risk letting the column's `NullCount` go out of sync. Thoughts? We _could_ potentially detect changes in the NullCount by going over each bit map after the yield, but that would be pretty inefficient.